go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/websocket v1.4.2
-	github.com/nats-io/nats.go v1.13.0
+	github.com/nats-io/nats.go v1.13.1-0.20220121202836-972a071d373d
```

Comment:
hmm why is this an untagged version? did you upgrade this, or upgrade NATS-server and this was a byproduct?

Reply:
I see. Could you use the instructions in [this PR](https://github.com/suborbital/grav/pull/42) to test with this branch to make sure it's all still working?

Reply:
Appears to work fine!